ddtrace/llmobs/_telemetry.py

```diff
@@ -0,0 +1,15 @@
+import time
+
+from ddtrace.internal.telemetry import telemetry_writer
+from ddtrace.internal.telemetry.constants import TELEMETRY_NAMESPACE
+
+
+def record_llmobs_enabled(error, agentless_enabled, site, start_ns):
+    tags = [("agentless", agentless_enabled), ("site", site), ("status", "error_{}".format(error) if error else "ok")]
```

Comment:
we should have a separate error tag so that we can set a monitor on there being any error rather than having to try to sum the different status values